addons/website_forum/views/website_forum.xml

```diff
@@ -686,437 +970,553 @@
+                <label for="filter" class="my-1 mr-2">Show Tags Starting By</label>
+                <select name="filter" class="custom-select" onchange="location = this.value;">
+                    <t t-foreach="pager_tag_chars" t-as="tuple_char">
+                        <option t-if="active_char_tag == tuple_char[1]" selected="true" value=""><t t-esc="tuple_char[0]"/></option>
+                        <option t-else="" t-attf-value="/forum/#{slug(forum)}/tag/#{quote_plus(tuple_char[1])}"><t t-esc="tuple_char[0]"/></option>
+                    </t>
+                </select>
+            </div>
         </nav>
-        <div class="row">
+
+        <div class="row mb-5" t-if="tags">
             <div class="col-md-3 mt16 o_js_forum_tag_follow" t-foreach="tags" t-as="tag">
-                <b><t t-esc="tag.posts_count"/>&amp;nbsp;</b>
-                <a t-attf-href="/forum/#{ slug(forum) }/tag/#{ slug(tag) }/questions?{{ keep_query( filters='tag') }}"
-                        t-attf-class="badge #{'badge-primary' if tag.message_is_follower else 'badge-secondary'} px-2">
-                    <span t-field="tag.name"/>
-                </a>
-                <span><i t-attf-class="fa #{'fa-star text-primary' if tag.message_is_follower else 'fa-star-o text-secondary'}" title="Subscribe" role="img" aria-label="Close"/></span>
-                <div t-attf-class="o_forum_tag_follow_box">
-                    <div class="input-group">
-                        <t t-call="website_mail.follow">
-                            <t t-set="email" t-value="user_id.email"/>
-                            <t t-set="object" t-value="tag"/>
-                        </t>
+                <span t-attf-class="badge  border #{'border-success text-success' if tag.message_is_follower else 'badge-light border text-600 '}   px-2">
+                    <i class="fa fa-tag small"/>
+                    <t t-esc="tag.name"></t>
+                    <b class="small align-top">(<t t-esc="tag.posts_count"/>)</b>
+                </span>
+                <div class="o_forum_tag_follow_box text-center">
+                    <div class="card shadow mt-2">
+                        <a t-attf-href="/forum/#{ slug(forum) }/tag/#{ slug(tag) }/questions?{{ keep_query( filters='tag') }}"
+                            class="btn btn-light">
+                            See <t t-esc="tag.posts_count"/> post<t t-if="tag.posts_count > 1">s</t>
+                        </a>
+                        <em class="d-block mb-2">or</em>
+                        <div class="input-group">
+                            <t t-call="website_mail.follow">
+                                <t t-set="email" t-value="user_id.email"/>
+                                <t t-set="object" t-value="tag"/>
+                            </t>
+                        </div>
                     </div>
                 </div>
             </div>
         </div>
+        <div t-else="" class="alert border text-center">
```

Comment:
alert border ?

Reply:
Yes, an alert with a border... what's the problem?

Reply:
Mmh I guess there is no equivalent... But here if a theme chooses to override the `$alert-border-width` variable, it will have no effect on those alert :/ But it feels like a bootstrap error to not use `$border-color` as default border color for alerts (and mentioning in their doc that `alert-*` classes are mandatory).

Reply:
even if you  change `$alert-border-width`, it's the default color (transparent) that determinate the visual appearance, and there is no variable controlling it.

> (and mentioning in their doc that alert-* classes are mandatory).

These classes are not mandatory, think about a  pure white alert over a dark bg, it make sense

Reply:
> These classes are not mandatory

I am never lying :p 
![image](https://user-images.githubusercontent.com/10338094/64713118-a2190680-d4bc-11e9-8b6b-201e4b7df6fe.png)

> it's the default color (transparent) that determinate the visual appearance, and there is no variable controlling it

Yes that was my point with "But it feels like a bootstrap error to not use $border-color as default border color for alerts" ;)